CHANGELOG.md

```diff
@@ -14,6 +14,54 @@
 
 -->
 
+# [2021-06-08]
+
+## Release Notes
+
+Deploy brig before galley (#1526, #1549)
```

Comment:
This comment is wire-cloud specific and doesn't apply to kubernetes installations (as there is no means to control the order of deployment for kubernetes installations). Maybe you could slightly adjust the wording?

Reply:
Yes, sure. https://github.com/wireapp/wire-server/pull/1577/commits/ee26eb4e9eebcc0f0c5630fc469089f0aa55fe65